README.md

```diff
@@ -17,6 +17,7 @@ The Unity Buy SDK allows Unity developers to query and sell products from Shopif
 - Create a cart
 - Check out via weblink
 - Checkouts via Apple Pay and Safari Web View
+- Checkouts via Chrome Custom Tab
```

Comment:
`Chrome Custom Tab` is just the name of the library we use as such as `Safari Web View`. We may want to provide images, but I think it could shown better in an example